BlazarUI/app/scripts/components/shared/RepoBranchCard.jsx

```diff
@@ -1,322 +1,13 @@
-import React, { Component, PropTypes } from 'react';
-import classNames from 'classnames';
-import { Link } from 'react-router';
-import { buildResultIcon } from '../Helpers';
-import moment from 'moment';
-import humanizeDuration from 'humanize-duration';
+import React, {Component} from 'react';
+import {Card} from 'card-stack-test';
 
-import Card from './Card.jsx';
-import ModuleRow from './ModuleRow.jsx';
-import BuildStates from '../../constants/BuildStates';
-import Loader from '../shared/Loader.jsx';
-import Sha from '../shared/Sha.jsx';
+import RepoBranchCardSummary from '../shared/RepoBranchCardSummary.jsx';
+import RepoBranchCardDetails from '../shared/RepoBranchCardDetails.jsx';
 
-class RepoBranchCard extends Card {
+function RepoBranchCard(props) {
+  const RenderedCard = Card(RepoBranchCardSummary, RepoBranchCardDetails);
```

Comment:
I'm thinking that it would make more sense to just directly pass in the react element instead of making Card something like a Component factory.
```
const summary = <RepoBranchCardSummary prop1={value1} prop2={value2} />
const details = <RepoBranchCardDetails prop={value}

return <Card summary={summary} details={details} />
```
Card shouldn't need to worry about how to pass in properties to the inner elements.